Library/Formula/mp3unicode.rb

```diff
@@ -0,0 +1,23 @@
+class Mp3unicode < Formula
+  desc "Command-line utility to convert mp3 tags between different encodings"
+  homepage "http://mp3unicode.sourceforge.net/"
+  url "https://github.com/downloads/alonbl/mp3unicode/mp3unicode-1.2.1.tar.bz2"
+  sha256 "375b432ce784407e74fceb055d115bf83b1bd04a83b95256171e1a36e00cfe07"
+  head "https://github.com/alonbl/mp3unicode.git"
+
+  depends_on "taglib"
+
+  def install
+    ENV.append "PKG_CONFIG", "true"
```

Comment:
Sorry, merged this but I'm also interested in the answer here and we can review in a future PR.